app/controllers/api/base_controller/normalizer.rb

```diff
@@ -20,37 +17,19 @@ def normalize_hash(type, obj, opts = {})
         end
 
         attrs.each do |k|
-          value =  normalize_direct(type, k, obj.kind_of?(ActiveRecord::Base) ? obj.try(k) : obj[k])
+          value = obj.kind_of?(ActiveRecord::Base) ? normalize_attr(k, obj.try(k)) : normalize_attr(k, obj[k])
```

Comment:
@imtayadeway liked this idea and played around with it quite a bit but ran into a few issues with simplifying.

When the `obj` is a hash for the actions and has a key `method`, it fails because `obj.try('method')` expects an input.

On the flipside, for an object like a `Tenant` , `obj['name']` returns a different value than `obj.try('name')` because of how defaults are set in the model, also causing a failure.